include/cantera/kinetics/Arrhenius.h

```diff
@@ -195,6 +198,97 @@ class ArrheniusRate final : public ArrheniusBase
 };
 
 
+//! Two temperature plasma reaction rate type depends on both
+//! gas temperature and electron temperature.
+/*!
+ * The form of the two temperature plasma reaction rate coefficient is similar to an
+ * Arrhenius reaction rate coefficient. The temperature exponent (b) is applied to
+ * the electron temperature instead. In addition, the exponential term with
+ * activation energy for electron is included.
+ *
+ *   \f[
+ *        k_f =  A T_e^b \exp (-Ea_T/RT) \exp (-Ea_{T_e}/RT_e)
+ *   \f]
```

Comment:
As both `Ea_T` and `Ea_Te` can have non-temperature units, I'm not sure about the subscripts 'T' here (it also appears that these are already replaced by later edits). Also, the LaTeX notation should likely use `E_{a,T}` instead of `Ea_T` (with 'T' being replaced by the eventual nomenclature).

Reply:
How about Ea(e) for electron activation energy?

Reply:
I'd work backwards from what makes most sense in terms of YAML notation (YAML field names will impose some constraints).

Reply:
It seems like def __cinit__(self, A=None, b=None, Ea=None, EE=None, also need to be the same as the key in yaml. This will restrict the name. Maybe EE is much easier compared to other option.

Reply:
What's the notation used in the literature for reactions that are parameterized like this? Can you share a link to any papers that use this parameterization for reactions so we can understand how this is typically written?

Reply:
Unfortunately, I didn't find any useful literature for naming. Most of the studies cite this paper https://iopscience.iop.org/article/10.1088/0963-0252/1/3/01, and Equation (45) (46) has the form.

Reply:
@BangShiuh  ... unfortunately the link doesn't work. Regardless, if there isn't a better convention in literature, your suggestion of `Ea`/`EE` should work well. As mentioned, having the ability to specify a parameter value that references temperature in the name with units of `cal/mol` (among others) may be somewhat confusing.

Reply:
I found that implementing a new name from YAML to python is still not very easy, because of the different accepted symbols.

Reply:
What about 
```yaml
equation: O2 + E <=> O2-
type: two-temperature-plasma
rate-constant: {A: 17283, b: -3.1, Ea: -700 K, EE: 1300 K}
```
which would be consistent with what you already have in C++?

Reply:
I discussed with Ray today, and we are thinking Ea-gas and Ea-electron which is more descriptive. There is no good literature for the name. But using hyphen requires using replace("-", "_") which is very confusing to me. I couldn't get it to work.

Reply:
Ah - I understand now. In this case, you'd need `Ea-gas` in YAML, whereas for Python keywords you'd need `Ea_gas`. The hyphen/underscore is one of the less intuitive things when matching YAML field names with Python keywords.